File: kitten-house-care/src/test/java/org/jesperancinha/housing/converter/CareCenterConverterTest.java
package org.jesperancinha.housing.converter;

import org.jesperancinha.housing.data.CareCenterDto;
import org.jesperancinha.housing.model.CareCenter;
import org.junit.jupiter.api.Test;

import static org.assertj.core.api.Assertions.assertThat;
import static org.junit.jupiter.api.Assertions.*;

class CareCenterConverterTest {

    @Test
    void toDto() {
        final CareCenterDto careCenterDto = CareCenterConverter.toDto(new CareCenter());

        assertThat(careCenterDto).isNotNull();
    }
}

File: kitten-house-care/src/test/kotlin/org/jesperancinha/housing/converter/CareCenterConverterTest.kt
package org.jesperancinha.housing.converter

import io.kotest.matchers.nulls.shouldNotBeNull
import org.assertj.core.api.Assertions
import org.jesperancinha.housing.dao.CareCenter
import org.jesperancinha.housing.service.toDto
import org.junit.jupiter.api.Test

internal class CareCenterConverterTest {
    @Test
    fun `conversion to Dto should not be null`() {
        val careCenterDto = CareCenter().toDto()
        careCenterDto.shouldNotBeNull()
    }
}